src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/WindowsSettings.json

```diff
@@ -0,0 +1,1748 @@
+  },
+  {
+    "Name": "PinyinImeSettingsUdp",
+    "Area": "TimeAndLanguage",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:regionlanguage-chsime-pinyin-udp"
+  },
+  {
+    "Name": "WubiImeSettingsUdp",
+    "Area": "TimeAndLanguage",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:regionlanguage-chsime-wubi-udp"
+  },
+  {
+    "Name": "Quickime",
+    "Area": "TimeAndLanguage",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:regionlanguage-quickime"
+  },
+  {
+    "Name": "PinyinImeSettings",
+    "Area": "TimeAndLanguage",
+    "Type": "SettingsApp",
+    "Note": "NoteImePinyin",
+    "Command": "ms-settings:regionlanguage-chsime-pinyin"
+  },
+  {
+    "Name": "Speech",
+    "Area": "TimeAndLanguage",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:speech"
+  },
+  {
+    "Name": "WubiImeSettings",
+    "Area": "TimeAndLanguage",
+    "Type": "SettingsApp",
+    "Note": "NoteImeWubi",
+    "Command": "ms-settings:regionlanguage-chsime-wubi"
+  },
+  {
+    "Name": "Activation",
+    "Area": "UpdateAndSecurity",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:activation"
+  },
+  {
+    "Name": "Backup",
+    "Area": "UpdateAndSecurity",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:backup"
+  },
+  {
+    "Name": "DeliveryOptimization",
+    "Area": "UpdateAndSecurity",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:delivery-optimization"
+  },
+  {
+    "Name": "FindMyDevice",
+    "Area": "UpdateAndSecurity",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:findmydevice"
+  },
+  {
+    "Name": "ForDevelopers",
+    "Area": "UpdateAndSecurity",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:developers"
+  },
+  {
+    "Name": "Recovery",
+    "Area": "UpdateAndSecurity",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:recovery"
+  },
+  {
+    "Name": "Troubleshoot",
+    "Area": "UpdateAndSecurity",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:troubleshoot"
+  },
+  {
+    "Name": "WindowsSecurity",
+    "Area": "UpdateAndSecurity",
+    "Type": "SettingsApp",
+    "AltNames": [ "WindowsDefender", "Firewall", "Virus", "CoreIsolation", "SecurityProcessor", "IsolatedBrowsing", "ExploitProtection" ],
+    "Command": "ms-settings:windowsdefender"
+  },
+  {
+    "Name": "WindowsInsiderProgram",
+    "Area": "UpdateAndSecurity",
+    "Type": "SettingsApp",
+    "Note": "NoteEnrolledWIP",
+    "Command": "ms-settings:windowsinsider"
+  },
+  {
+    "Name": "WindowsInsiderProgramOptin",
+    "Area": "UpdateAndSecurity",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:windowsinsider-optin"
+  },
+  {
+    "Name": "WindowsUpdate",
+    "Area": "UpdateAndSecurity",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:windowsupdate"
+  },
+  {
+    "Name": "WindowsUpdateActions",
+    "Area": "UpdateAndSecurity",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:windowsupdate-action"
+  },
```

Comment:
**Adding settings element for `Optional Updates` page:**
```suggestion
},
{
  "Name": "WindowsUpdateViewOptionalUpdates",
  "Area": "UpdateAndSecurity",
  "Type": "SettingsApp",
  "AltNames": [ "WindowsUpdate" ],
  "IntroducedInVersion": 2004,
  "Command": "ms-settings:windowsupdate-optionalupdates"
},
```

Reply:
@crutkas, @TobiasSekan
When commiting this, be careful on the `IntroducedInVersion` value.

Reply:
Added